src/hidapi/SDL_hidapi.c

```diff
@@ -40,7 +40,7 @@
 
 #ifndef SDL_HIDAPI_DISABLED
 
-#if defined(SDL_PLATFORM_WIN32) || defined(SDL_PLATFORM_WINGDK)
+#if defined(SDL_PLATFORM_WINDOWS)
```

Comment:
WINGDK

Reply:
I was too quick on the push, this one got fixed in the followup commit 6b294a09b5dc7b2653e63a6de29bb88894918342.